docs/lib/datastore/fragments/android/getting-started/30_platformIntegration.md

```diff
@@ -13,7 +13,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.1'
+        classpath 'com.android.tools.build:gradle:4.1.2'
```

Comment:
Is this the guy the breaks the build or did they finally patch it?

Reply:
Not sure if the issue we observed when attempting to move to 4.1.2 for the library itself also happens when users are creating an app.

Reply:
This breaks the build _of our library itself_, but it should work fine for a consumer of our library. (I've been using it myself, in a sample app.)

Reply:
The issue we are running into with the library is trying to jump from 4.0.x to 4.1.x, not 4.1.1 to 4.1.2, so I think this change is safe.